### PR TITLE
add fallback for optional account

### PR DESCRIPTION
**Pull request by raintygao**
### Description
add some fallback as security is set optional

### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test.
- [ ] New functionality has user manual doc added.
- [ ] Commits are signed per the DCO using --signoff.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by ruanyl — APPROVED** [review]

File: public/components/chat_welcome_message.tsx [hunk 1]
```diff
@@ -19,7 +20,7 @@ export const WelcomeMessage = (props: Props) => {
       titleSize="s"
       body={
         <EuiText color="default">
-          <p>Welcome {props.username} to the OpenSearch Assistant!</p>
+          <p>Welcome{usernamePlaceHolder}to the OpenSearch Assistant!</p>
```
Comment (wanglam) [inline public/components/chat_welcome_message.tsx:23]: Shall we add empty space here? `Welcometo` would be weird.

Comment (raintygao) [reply public/components/chat_welcome_message.tsx:23]: Space is predefined in `usernamePlaceHolder`.

Comment (raintygao) [reply public/components/chat_welcome_message.tsx:23]: For username case, there is one space before and after the name, and only one space in the case without username.

Comment (raintygao) [reply public/components/chat_welcome_message.tsx:23]: Space for non username case has been added, you can check it again.

**Review by wanglam — APPROVED** [review]

**Pull request merged**